### ISSUE TITLE
AttributeError: 'CpModel' object has no attribute 'AddExactlyOne'

### ISSUE THREAD

**Issue by FishofJLlake** [google/or-tools#3094]
**What version of OR-Tools and what language are you using?**
Version: master
Language: python

**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
none
**What operating system (Linux, Windows, ...) and version?**
ubuntu20.04LTS
**What did you do?**
when I run 
`python3 ./examples/python/flexible_job_shop_sat.py `
then return an error：
`AttributeError: 'CpModel' object has no attribute 'AddExactlyOne'`
I also checked the cp_model source code, and I found it had 'AddExactlyOne', but why?

**Comment by FishofJLlake** [comment]
then I Modified the code：
`model.AddExactlyOne(l_presences)` to
`model.Add(sum(l_presences) == 1)`
run OK.

**Comment by lperron** [comment]
You have a mismatched versions. Addexactlyone has been added to master, but master is ahead of the latest release.  You need to rebuild from source to use it.

**Comment by tobiasBora** [comment]
I can’t find `Addexactlyone` documented in https://developers.google.com/optimization/reference/python/sat/python/cp_model, am I on the wrong doc? Where could I find an up to date documentation?

**Comment by lperron** [comment]
The web page is not up to date.
Go to
https://google.github.io/or-tools/python/ortools/sat/python/cp_model.html

**Comment by tobiasBora** [comment]
Great, thanks a lot!

**Comment by Mizux** [comment]
https://or-tools.github.io/docs/python/ortools/sat/python/cp_model.html#CpModel.AddExactlyOne ?

**Comment by tobiasBora** [comment]
Ahah, thanks for this other reference (seems like it’s actually the same documentation as the link of lperron, but on a third domain…). They should update  https://developers.google.com/optimization/reference/python/sat/python/cp_model however, especially when the same website refers to AddExactlyOne in the tutorials.

**Comment by Mizux** [comment]
TLDR: https://google.github.io/or-tools/ (ed we should kill this github-page to avoid confusion)
was the old doc generated (9.3) and pushed directly in google/or-tools repo BUT it generates thousands of files, take few hundreds mega, and windows couldn't clone anymore our repo since some path was more than 512 char `-_-`.

So, we migrate the generated doc to its own repo -> https://github.com/or-tools/docs with it own github-page https://or-tools.github.io/docs/ (this doc is generated from a script in google/or-tools -> you should be able to regenerate it locally if you want ;))

For the optimization  https://developers.google.com/optimization/reference/python/sat/python/ it is generated by a script which is borken so we need to fix to upgrade this reference doc OR we should remove it and directly point it to the link above: https://github.com/or-tools/docs aka one source of truth...

**Comment by sharanyaawasthi** [comment]
> The web page is not up to date.
> Go to
> https://google.github.io/or-tools/python/ortools/sat/python/cp_model.html
> 
> Le lun. 15 mai 2023, 20:45, Tobias Bora ***@***.***> a écrit :
> […](#)

Thank you! I was pretty confused as well as I couldn't find AddExactlyOne().
